src/ring_theory/adjoin_root.lean

```diff
@@ -120,7 +120,7 @@ def amk : polynomial α →ₐ[α] adjoin_root f :=
 
 lemma amk_eq_mk : ⇑(amk f) = mk := rfl
 
-variables (a : β) (h : f.eval₂ (algebra_map β) a = 0)
+variables (a : β) (h : aeval α β a f = 0)
```

Comment:
From the point of view of "meaningful" variable names, I would prefer if `a` were renamed to `b`. Otherwise I'm sure that future me will assume that the type of `a` is `alpha` instead of `beta` when reading some of the proofs below.